src/cmds/core/other.py

```diff
@@ -25,7 +25,15 @@ async def no_hints(
             "Once the event is over you are more then welcome to share solutions/write-ups/etc and try them in the "
             "After Party event."
         )
-
+    @slash_command(guild_ids=settings.guild_ids, description="A simple reply proving a link to the support desk article on how to get support")
+    @cooldown(1, 60, commands.BucketType.user)
```

Comment:
Use `@commands.cooldown` here since we are not explicitly importing the `cooldown` decorator

```suggestion
    @commands.cooldown(1, 60, commands.BucketType.user)
```